frontend/components/DocumentEditToolbar/DocumentEditToolbar.jsx

```diff
@@ -111,7 +117,7 @@ export default class DocumentListToolbar extends Component {
         <div class={styles['padded-group']}>
           <ButtonWithOptions
             accent="save"
-            disabled={disabled}
+            disabled={hasValidationErrors || disabled}
```

Comment:
I'd prefer we're as explicit about connection errors as we are about validation. `disabled` should be `hasConnectionErrors`